subscription-manager.spec

```diff
@@ -118,11 +118,8 @@ from the server. Populates /etc/docker/certs.d appropriately.
 %{_datadir}/rhsm/subscription_manager/plugin/container.py*
 # Copying Red Hat CA cert into each directory:
 %attr(755,root,root) %dir %{_sysconfdir}/docker/certs.d/cdn.redhat.com
-%attr(755,root,root) %dir %{_sysconfdir}/docker/certs.d/access.redhat.com
-%attr(755,root,root) %dir %{_sysconfdir}/docker/certs.d/registry.access.redhat.com
-%attr(644,root,root) %{_sysconfdir}/docker/certs.d/cdn.redhat.com/redhat-uep.crt
-%attr(644,root,root) %{_sysconfdir}/docker/certs.d/access.redhat.com/redhat-uep.crt
-%attr(644,root,root) %{_sysconfdir}/docker/certs.d/registry.access.redhat.com/redhat-uep.crt
+%attr(644,root,root) %{_sysconfdir}/rhsm/ca/redhat-entitlement-authority.pem
```

Comment:
Will having that cert in /etc//rhsm/ca/redhat-entitlement-authority.pem  and as part of the bundle in /etc/rhsm/ca/redhat-uep.pem cause any problems?

I think our client should be okay, though I wonder if it could cause oddities with NSS/GNUTLS (ie, yum or ostree).  (The content plugins seem to always point to /etc/rhsm/ca/redhat-uep.pem specifically, so shouldn't be loading anything else)